docs/TREEDB_C8_DEFAULT_PROMOTION_2974.md

```
@@ -0,0 +1,142 @@
+# TreeDB c8 default promotion (#2974)
+
+This is the closeout report for the post-#2960 span-native apply default graph:
+#2975, #2976, #2977, #2978, #2979, and #2980.
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)